ivy/stateful/module.py

```diff
@@ -1141,48 +1153,468 @@ def build_mode(self):
         return self._build_mode
 
     @property
-    def built(self):
+    def built_(self):
         return self._built
 
-    # Methods #
-    # ------- #
+    # Module Converters #
+    # ------------------#
+    
+    @staticmethod
+    def static_to_haiku_module(ivy_module):
```

Comment:
no need to have a `static_to_haiku_module`, you can just have all of this logic implemented in the instance method `to_haiku_module`. The same goes for all other instance methods.